packages/tocco-ui/src/Panel/Panel.js

```diff
@@ -0,0 +1,57 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+
+/**
+ * <Panel/> is used to conceal and display related content alternating by interaction.
+ */
+class Panel extends React.Component {
+  state = {
```

Comment:
does this really work? usually the state is set with this.setState

Reply:
Yes, because of babel. I do that according a react course I took.

* https://stackoverflow.com/questions/37788342/is-it-better-to-define-state-in-constructor-or-using-property-initializers
* https://stackoverflow.com/questions/43546575/react-defining-state-without-a-constructor?rq=1
* https://stackoverflow.com/questions/42993989/init-state-without-constructor-in-react

Reply:
I am open for coding style.

Reply:
ok. just didnt know this notation